apps/hash-graph/lib/graph/src/store/postgres.rs

```diff
@@ -263,21 +262,21 @@ where
                 FROM create_owned_ontology_id(
                     base_uri := $1,
                     version := $2,
-                    owned_by_id := $3,
-                    record_created_by_id := $4
+                    record_created_by_id := $4,
+                    owned_by_id := $3
                 );"#,
                 &[
                     &metadata.record_id().base_uri.as_str(),
                     &metadata.record_id().version,
-                    &metadata.owned_by_id(),
                     &metadata.provenance_metadata().updated_by_id(),
+                    &metadata.owned_by_id(),
```

Comment:
nit: 
Should we swap these around so they're in order with respect to the params in `create_owned_ontology_id`? It's strange to see the order being `1,2,4,3` above. I suppose this is the case for the calls function below in this file as well